src/main/java/com/bandwidth/sdk/BandwidthClient.java

```diff
@@ -73,7 +91,17 @@ public synchronized static BandwidthClient getInstance() {
                 apiVersion = System.getProperty(BandwidthConstants.BANDWIDTH_SYSPROP_API_VERSION);
             }
 
-            INSTANCE = new BandwidthClient(userId, apiToken, apiSecret, apiEndpoint, apiVersion);
+            if (maxTotal == null) {
+                maxTotal = System.getProperty(BandwidthConstants.BANDWIDTH_SYSPROP_HTTP_MAX_TOTAL_CONNECTIONS,
+                                              BandwidthConstants.HTTP_MAX_TOTAL_CONNECTIONS);
+            }
+
+            if (defaultMaxPerRoute == null) {
+                defaultMaxPerRoute = System.getProperty(BandwidthConstants.BANDWIDTH_SYSPROP_HTTP_MAX_DEFAULT_CONNECTIONS_PER_ROUTE,
+                                                        BandwidthConstants.HTTP_MAX_DEFAULT_CONNECTIONS_PER_ROUTE);
+            }
+
+            INSTANCE = new BandwidthClient(userId, apiToken, apiSecret, apiEndpoint, apiVersion, maxTotal, defaultMaxPerRoute);
```

Comment:
It is up to you, but I would change to use a class to carry those values instead of passing them as parameter